pages/linux/eselect.md

```diff
@@ -0,0 +1,17 @@
+# eselect
+
+> Gentoo's multi-purpose configuration and management tool.
+> It consists of various modules that take care of individual administrative tasks.
+> More information: <https://wiki.gentoo.org/wiki/Eselect>.
+
+- Display a list of installed modules:
+
+`eselect`
+
+- View documentation for a specific module:
```

Comment:
```suggestion
- View the tldr page of a specific module:
```

Maybe like that, since our pages are not official documentation? :)

Reply:
All pages that redirect to other pages use that same description, at least from what I've seen ([nmcli for example](https://github.com/tldr-pages/tldr/blob/main/pages/linux/nmcli.md)). It's also in the style guide for alias pages.

Reply:
Good, in this case my comment is void